app/Handlers/Events/SlackSubscriber.php

```diff
@@ -6,8 +6,11 @@ class SlackSubscriber
 {
     public function subscribe($events)
     {
-        $events->listen('new-signup', [$this, 'onNewSignup']);
-        $events->listen('new-conference', [$this, 'onNewConference']);
+        $endpoint = Slack::getEndpoint();
```

Comment:
Think you mean `empty()` versus `!empty()` (i.e. moving the body of the `if` branch outside of it to `return` early requires negating the original branch condition). That said, fair enough. Fixed.